browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -172,34 +175,56 @@ BraveBrowserView::BraveBrowserView(std::unique_ptr<Browser> browser)
                           base::Unretained(this)));
 #endif
 
+  const bool can_have_vertical_tabs =
+      browser_->is_type_normal() &&
+      base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs);
+  const bool can_have_sidebar = sidebar::CanUseSidebar(browser_.get());
+  bool need_to_wrap_contents_container = can_have_vertical_tabs;
 #if BUILDFLAG(ENABLE_SIDEBAR)
   // Only normal window (tabbed) should have sidebar.
-  if (!sidebar::CanUseSidebar(browser_.get())) {
+  need_to_wrap_contents_container =
+      need_to_wrap_contents_container || can_have_sidebar;
+#endif  // BUILDFLAG(ENABLE_SIDEBAR)
+
+  if (!need_to_wrap_contents_container)
```

Comment:
the buildflag guard for `ENABLE_SIDEBAR` should end after this return

Reply:
actually ENABLE_SIDEBAR should just go away and sidebar should move into ui/views, but that's a pretty big change so let's just make sure the guards are correct in this PR